tests/test_loadflow.py

```diff
@@ -120,6 +120,40 @@ def test_validation_all():
     assert validation.twts is not None
 
 
+def test_lf_validation_parameters():
+    # Testing setting independently every attributes
+    attributes = {
+        'threshold': [0.1, 0.2],
+        'verbose': [True, False],
+        'loadflow_name': ['loadFlow1', 'loadFLow2'],
+        'epsilon_x': [0.1, 0.2],
+        'apply_reactance_correction': [True, False],
+        'ok_missing_values': [True, False],
+        'no_requirement_if_reactive_bound_inversion': [True, False],
+        'compare_results': [True, False],
+        'check_main_component_only': [True, False],
+        'no_requirement_if_setpoint_outside_power_bounds': [True, False]
+    }
+
+    for attribute, values in attributes.items():
+        for value in values:
+            parameters = lf.ValidationParameters(**dict([(attribute, value)]))
+            assert value == getattr(parameters, attribute)
+
+            parameters = lf.ValidationParameters()
+            setattr(parameters, attribute, value)
+            assert value == getattr(parameters, attribute)
+
+    n = pp.network.create_four_substations_node_breaker_network()
```

Comment:
I would advise to separate in 2 tests the getters/setters and the test of the behaviour below